src/lib/userStorage/UserStorageClass.js

```diff
@@ -518,6 +519,7 @@ export class UserStorage {
     const seed = this.wallet.wallet.eth.accounts.wallet[this.wallet.getAccountForType('gundb')].privateKey.slice(2)
     await this.feedDB.init(seed, this.wallet.getAccountForType('gundb')) //only once user is registered he has access to realmdb via signed jwt
     await this.initFeed()
+    await this.initRealmDBUserStorage()
```

Comment:
dont use realmdb in names, the userstorge only knows it is using a profilestorage api